lib/annotations.go

```diff
@@ -0,0 +1,35 @@
+package metadata
+
+const (
+	// CheckpointAnnotationContainerManager is used when creating an OCI image
+	// from a checkpoint archive to specify the name of container manager.
+	CheckpointAnnotationContainerManager = "org.criu.checkpoint.container.manager"
+
+	// CheckpointAnnotationName is used when creating an OCI image
+	// from a checkpoint archive to specify the name of the checkpoint.
+	CheckpointAnnotationName = "org.criu.checkpoint.container"
```

Comment:
"container" could mean many different things.
What about `"org.criu.checkpoint.name"`?

Reply:
Yes you are right

Reply:
@rst0git Won't it be misunderstood as the name of the checkpoint? Here, this annotation is used to specify the container name.
should we go with - `org.criu.checkpoint.container.name`

Reply:
> should we go with - org.criu.checkpoint.container.name

Sounds good